src/test/java/org/salesforce/steps/featureSteps.java

```diff
@@ -5,9 +5,11 @@
 import io.cucumber.java.en.Given;
 import io.cucumber.java.en.Then;
 import io.cucumber.java.en.When;
+
```

Comment:
You didnt ordered the imports according to the checkstyle and team conventions

Reply:
Fix this problem is part of other member of team